tests/integration/fixtures/bacnet-network/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:20.04 as base
+
+WORKDIR /opt
+
+RUN apt update \
+    && apt install -y \
+    python3 \
+    python3-pip \
+    && rm -rf /var/lib/apt/lists/*
+
+COPY requirements.txt .
```

Comment:
it doesn't look like you're ever installing the `requirements.txt`

Reply:
added! Looks like there a bunch of changes that I had on the wrong machine that never got pushed. I'll fix that today